mods/persistence/modules/world_save/_persistence.dm

```diff
@@ -27,13 +27,23 @@ SAVED_VAR(/datum, custom_saved)
 	custom_saved = null //Clear it since its no longer needed
 #endif
 
-/** 
+/**
  * Called immediately after the datum has been loaded from save during SSMapping's init, and before Initialize().
  * DO NOT call anything relying on subsystems being initialized in this!!
-*/ 
+*/
 /datum/proc/after_deserialize()
 	return
 
 /**Used to check and override whether an entity should be saved. */
 /datum/proc/should_save()
 	return should_save
+
+//**Used to check if a movable will be saved by world save. */
+/atom/movable/proc/will_save()
```

Comment:
What is that proc for? It's a bit awkwardly named, and its description make it sound like it's identical to what should_save() ?
Like what's the difference between that, and should_save()?

Reply:
This proc determines whether or not an atom will actually be saved during world save iteration, meaning it's either in a saved z-level or saved area. For example it is now used to determine if a mob will be saved by the world save or if it needs to be saved in limbo, and also used to determine if extensions should self-report for saving depending on whether or not their holder will save.

Reply:
Alright, but can you put that in the comments and give it a name that's a bit more representative of that? Like maybe "should_save_in_limbo", or something else?